tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+- name: Ensure download location exists
```

Comment:
It occurred to me just now that I didn't implement the `become` vars at all, so I'll get that added shortly. :)